ui/app/src/components/PublishDialog/PublishDialogForm.tsx

```diff
@@ -219,7 +225,7 @@ export function PublishDialogForm(props: PublishFormProps) {
   return (
     <form className={classes.root}>
       <section className={classes.checkboxes}>
-        {showRequestApproval && (
+        {showRequestApproval && published && (
```

Comment:
I don't remember `published` being related to showing request approval or the email approval.

Reply:
can you confirm this with sumer? I remember that the first publish cant be sent for approval